src/components/Kv/KvCurrencyInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<input
```

Comment:
I think we'll need to add "id" as a required prop and use it on this input here so that <label for="my-cool-id"> outside of this component point to the input.

Reply:
agree

Reply:
This is already happening. The id from here -> `<kv-currency-input id="amount"` gets rendered unto the input element